src/apps/routes/webhooks.clj

```diff
@@ -26,6 +26,14 @@
                          :summary "List notification topics"
                          :description "Returns all of the notification topics defined"
                          (ok (list-topics)))
+                    )
+           (context "/types" []
+                    (GET "/" []
+                         :query [params SecuredQueryParams]
+                         :return WebhookTypeList
+                         :summary "List webhooks types"
+                         :description "Returns all webhook types defined"
+                         (ok (list-types)))
                     ))
```

Comment:
This is so minor that I feel bad mentioning it, but the convention for closing parentheses in Clojure is to put them at the end of the form rather than to put them on a line by themselves.

Reply:
👍